Build/GameSettings.h

```diff
@@ -57,6 +58,7 @@ enum
 struct GAME_SETTINGS
 {
 	INT8				bLastSavedGameSlot;							// The last saved game number goes in here
+	wchar_t     sCurrentSavedGameName[SIZE_OF_SAVE_GAME_DESC];                     // The name of the current Savegame goes here. Only relevant for Dead is Dead games
```

Comment:
Can I safely use UTF8 here?
This is a temporary variable that it used to copy data from the savegame header, which is wchar_t at the moment. I fear to break stuff here.